File: College/C/Exercises/1ª Lista de Exercícios (1S)/Exercício02.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    float n1, n2, n3, media;

    printf("Digite o primeiro número: ");
    scanf("%f", &n1);

    printf("Digite o segundo número: ");
    scanf("%f", &n2);

    printf("Digite o terceiro número: ");
    scanf("%f", &n3);

    media = (n1 + n2 + n3) / 3;

    printf("A média entre %f, %f, %f é igual a %f\n", n1, n2, n3, media);

}

File: Java/1ª Lista de Exercícios (1S)/Exercício02/Exercício02.java
import java.util.Scanner;

public class Exercício02{
	public static void main(String args[]){
		Scanner scan = new Scanner(System.in);

		System.out.print("Digite o primeiro número: ");
		int n1 = scan.nextInt();
	
		System.out.print("Digite o segundo número: ");
		int n2 = scan.nextInt();

		System.out.print("Digite o terceiro número: ");
		int n3 = scan.nextInt();

		float m = (n1 + n2 + n3) / 3;

		System.out.println("A média entre os números é igual a " + m);

		scan.close();
	}
}
